### PR TITLE
feat: add Enkrypt

### PR DESCRIPTION
**Pull request by kvhnuke**
## Description

add Enkrypt injected wallet detection

### CODE REVIEW

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a patch changeset (`pnpm changeset`) and add a test case for `getInjectedName.ts`

**Comment by kvhnuke** [comment]
@tmm please let me know if you need any other changes

File: .changeset/seven-bottles-film.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/connectors': minor
+---
+
+Add Enkrypt extension detection
```
Comment (tmm) [inline .changeset/seven-bottles-film.md:5]:
```suggestion
Added Enkrypt extension detection
```

File: .changeset/seven-bottles-film.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/connectors': minor
```
Comment (tmm) [inline .changeset/seven-bottles-film.md:2]:
```suggestion
'@wagmi/connectors': patch
```

**Review by tmm — APPROVED** [review]

**Pull request merged**